src/main/scala/fi/oph/koski/massaluovutus/valintalaskenta/ValintalaskentaResponse.scala

```diff
@@ -0,0 +1,155 @@
+package fi.oph.koski.massaluovutus.valintalaskenta
+
+import fi.oph.koski.schema._
+import fi.oph.koski.suoritusjako.suoritetuttutkinnot.SuoritetutTutkinnotAmmatillisenTutkinnonOsittainenSuoritus
+
+import java.time.{LocalDate, LocalDateTime}
+
+case class ValintalaskentaResult(
+  oppijaOid: String,
+  opiskeluoikeudet: Seq[ValintalaskentaOpiskeluoikeus],
+)
+
+case class ValintalaskentaOpiskeluoikeus(
+  tyyppi: Koodistokoodiviite,
+  oid: String,
+  versionumero: Int,
+  aikaleima: LocalDateTime,
+  oppilaitos: Option[Oppilaitos],
+  tila: ValintalaskentaOpiskeluoikeudenTila,
+  suoritukset: Seq[ValintalaskentaSuoritus],
+)
+
+object ValintalaskentaOpiskeluoikeus {
+  def apply(oo: KoskeenTallennettavaOpiskeluoikeus): ValintalaskentaOpiskeluoikeus = ValintalaskentaOpiskeluoikeus(
```

Comment:
Pohdin, olisiko tämä parempi tehdä sillä Kela- yms. servlettien käyttämällä "deserialisoi rajoitettuun skeemaan" -tavalla kenttä kerrallaan konvertoinnin sijasta, jos palautettava data on kuitenkin subset opiskeluoikeus-skeemasta ja filtteröintikin perustuu vain opiskeluoikeus- ja suoritustyyppeihin?

Reply:
Tässä ollaan niin pienen palautettavan subsetin kanssa tekemisissä että itse koin tämän paremmaksi, kun tarvittavaa mäppäyskoodia tulee joka tapauksessa niin vähän. Skeema toki on subsetti master-skeemasta näin aivan ekassa versiossa, mutta varsinaista tarvetta tällaiselle rajoitukselle ei ole.